src/fiatconnect/slice.ts

```diff
@@ -47,11 +48,19 @@ export interface CachedQuoteParams {
   fiatType: FiatType
 }
 
+export interface CachedTransferDetails {
+  transferId: string
+  providerId: string
+  fiatAccountId: string
+  quote: QuoteResponse['quote']
```

Comment:
This `quote` is typed to pull the `quote` object out of the [quote response schema](https://github.com/fiatconnect/fiatconnect-types/blob/main/src/quote.ts#L117), right? Why don't we store the entire quote (i.e., the `kyc` and `fiatAccount` objects as well)?